### ISSUE TITLE
OneDrive: unable to uninstall

### ISSUE THREAD

**Issue by xceptic-io** [10se1ucgo/DisableWinTracking#335]
**INFORMATION:**

* Windows Edition (Home/Pro/Enterprise):
* Architecture (32bit/64bit):
* Options used:

**LOG:**

```python
REPLACE THIS WITH DWT.LOG
```

**Comment by xceptic-io** [comment]
OneDrive: unable to uninstall

**Comment by jriggz** [comment]
I've encountered the same issue; Windows 10 Enterprise, 1703 x64.
"08:54:12 INFO: OneDrive: unable to uninstall"
[dwt-2017-05-22.txt](https://github.com/10se1ucgo/DisableWinTracking/files/1019477/dwt-2017-05-22.txt)

**Comment by phpchess** [comment]
10:45:12 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
10:45:12 INFO: ('Windows', 'zzzzzzzzz', '10', '10.0.15063', 'AMD64', 'Intel64 Family 6 Model 42 Stepping 7, GenuineIntel')
10:45:12 INFO: DisableWinTracking version 3.1.2
10:46:13 INFO: IP Blocker: The IP 134.170.30.202 was successfully blocked.
10:46:13 INFO: IP Blocker: The IP 137.116.81.24 was successfully blocked.
10:46:13 INFO: IP Blocker: The IP 157.56.106.189 was successfully blocked.
10:46:13 INFO: IP Blocker: The IP 2.22.61.43 was successfully blocked.
10:46:13 INFO: IP Blocker: The IP 2.22.61.66 was successfully blocked.
10:46:13 INFO: IP Blocker: The IP 204.79.197.200 was successfully blocked.
10:46:13 INFO: IP Blocker: The IP 23.218.212.69 was successfully blocked.
10:46:13 INFO: IP Blocker: The IP 64.4.54.254 was successfully blocked.
10:46:14 INFO: IP Blocker: The IP 65.39.117.230 was successfully blocked.
10:46:14 INFO: IP Blocker: The IP 65.52.108.33 was successfully blocked.
10:46:14 INFO: IP Blocker: The IP 65.55.108.23 was successfully blocked.
10:46:14 INFO: DiagTrack: Take Ownership of AutoLogger-Diagtrack-Listener.etl was successful
10:46:14 ERROR: 1
None
10:46:14 INFO: DiagTrack: Grant Admin Privilege of AutoLogger-Diagtrack-Listener.etl was successful
10:46:14 ERROR: 2
None
10:46:14 INFO: DiagTrack: Cleared AutoLogger-Diagtrack-Listener.etl
10:46:14 INFO: DiagTrack: Deny System Privilege of AutoLogger-Diagtrack-Listener.etl was successful
10:46:14 INFO: DiagTrack: Successfully cleared and locked DiagTrack log.
10:46:14 INFO: Services: Succesfully removed service 'dmwappushsvc'
10:46:14 INFO: Services: Succesfully removed service 'DiagTrack'
10:46:14 INFO: Registry: Successfully modified dmwappushsvc key.
10:46:14 INFO: Registry: Successfully modified DiagTrack key.
10:46:14 INFO: Registry: Successfully modified AllowTelemetry key.
10:46:14 ERROR: Hosts: Failed to modify hosts file
Traceback (most recent call last):
  File "dwt_util.py", line 244, in host_file
IOError: [Errno 13] Permission denied: 'C:\\WINDOWS\\System32/drivers/etc/hosts'
10:46:14 INFO: Registry: Successfully modified ListPin key.
10:46:14 INFO: Registry: Successfully modified FileSync key.
10:46:14 INFO: OneDrive: unable to uninstall
10:46:14 INFO: Done. It's recommended that you reboot as soon as possible for the full effect.

**Comment by jriggz** [comment]
It is an issue, though. It won't uninstall OneDrive, which is functionality it's suppose to have. Why is this issue "invalid?"

**Comment by Ruined1** [comment]
@jriggz it was marked "invalid" because you didn't follow the format at all. You didn't post your operating system... You didn't post a log... You didn't do anything except put a title.

This issue has been resolved in the [latest version.](https://github.com/10se1ucgo/DisableWinTracking/releases/tag/v3.2.0)

**Comment by jriggz** [comment]
@Ruined1 Really?

“I've encountered the same issue; Windows 10 Enterprise, 1703 x64.
"08:54:12 INFO: OneDrive: unable to uninstall"”

And then the log was posted as an attachment. I absolutely followed the format, thanks, though.

**Comment by Ruined1** [comment]
@jriggz Yes, really. I had to catch up on about 100 issues because I am the only one maintaining this project anymore. The last commit not by me, was April 4, 2017. The last commit by the original developer was November 17, 2016. Most developers aren't going to look past your first post ( which had you followed the format, wouldn't look like [this](https://i.imgur.com/oZpuBiZ.png) ) if it looks completely default.

Anyways, the issue was resolved. Next time you submit an issue, follow the original template provided to you by the developer. It's there for a reason. Thank you for your contribution.
